ttnn/cpp/ttnn/operations/ccl/all_gather/device/all_gather_op.cpp

```diff
@@ -62,7 +62,7 @@ AllGatherBidirectionalMode AllGatherConfig::choose_bidirectional_mode(Tensor con
     }
 
     std::size_t eth_l1_capacity = eth_l1_mem::address_map::MAX_L1_LOADING_SIZE - eth_l1_mem::address_map::ERISC_L1_UNRESERVED_BASE;
-    std::size_t tensor_size_bytes = input_tensor.shape().volume() * input_tensor.element_size();
+    std::size_t tensor_size_bytes = input_tensor.intended_volume() * input_tensor.element_size();
```

Comment:
To me it seems like this should be .volume() call, but I did not want to trigger any changes.

CC @SeanNijjar

Reply:
also see
https://github.com/tenstorrent/tt-metal/pull/13372/files#r1784806467

Reply:
Discussed offline - `intended_volume` is what we're ideally looking for here. Thanks @ayerofieiev-tt